gradle.properties

```
@@ -47,7 +47,7 @@ stdlibAuthVersion=2.8.0
 stdlibJwtVersion=2.8.0
 stdlibOAuth2Version=2.8.0
 
-stdlibHttpVersion=2.8.1
+stdlibHttpVersion=2.8.3-20231012-153400-b73fc00
```

Need to update the version after http release
```suggestion
stdlibHttpVersion=2.8.3
```